python/py-fasttext/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        facebookresearch fastText 0.9.2 v
+revision            0
+name                py-[string tolower ${github.project}]
+
+categories-append   textproc
+platforms           darwin
+license             MIT
+maintainers         nomaintainer
+platforms           darwin
+
+description         Library for fast text representation and classification.
+
+long_description    fastText is a library for efficient learning of \
+                    word representations and sentence classification.
+
+checksums           rmd160  f83305b9e95c7e97a0a51c8a82293b235346315b \
+                    sha256  e9ce4ab7d2bf6c5e2343faa5920c59370d3e8a08bf6bd07b1f1e7ea48d441fe7 \
+                    size    4036928
+
+python.versions     38 39
+
```

Comment:
it looks like the port requires c++11 or c++14 (and will pick the latter if available); so I guess you should request a compiler that is compatible with this?

Reply:
Thanks done.